### DISCUSSION TITLE
remark without package.json

### DISCUSSION THREAD

**Discussion by sarmong** [remarkjs/remark#930] [Q&A] [answered]
Can I use remark on non-javascript project?

I can install it globally, however, plugins seem to work only when installed in a project.

I want to setup `remark-language-server` to be used with neovim, so that it can parse any markdown file regardless of what is installed in a project.

**Comment by wooorm** [comment, marked as answer]
Yes, you can use it on a project in another language, but indeed, `remark` and its ecosystem are in JavaScript. You should use a `package.json` with them. That way you can version them. Otherwise stuff will randomly break

**Reply by sarmong** [reply]
>Yes, you can use it on a project in another language

But only core remark, right? Plugins are impossible to use without?

**Reply by wooorm** [reply]
I mean that, when you have a project that includes python code, you can still `npm install remark remark-preset-whatever`
